examples/app-python/README.md

```diff
@@ -28,6 +37,41 @@ The sequences supported by the example are described below:
 * `HelloFriend`: accepts parameter, logging "hello, <parameter>!"
 * `LogLocation`: queries the current satellite location and logs it
 
+## Working with I/O interface
+
+# Configuring GPIO in application
```

Comment:
Should be `##` not `#`

Reply:
Same for all other following headings.

Reply:
Updated headings and subheadings. Would you check if they are OK now?